cscs-checks/mch/collectives_halo.py

```diff
@@ -102,13 +105,14 @@ def __init__(self, variant):
                                  'default': 0.0138493
                              }
                          })
+        self.descr = 'Alltoall communication; Meteoswiss check'
```

Comment:
I don't see the reason for adding the `Meteoswiss check` annotation. We do have the `mch` tag already.